src/main/java/org/fundacionjala/pivotal/cucumber/selenium/menu/ToolBars.java

```diff
@@ -0,0 +1,113 @@
+package org.fundacionjala.pivotal.cucumber.selenium.menu;
+
+
+import java.util.concurrent.TimeUnit;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.fundacionjala.pivotal.cucumber.selenium.browser.DriverManager;
+import org.fundacionjala.pivotal.cucumber.selenium.pages.AbstractBasePage;
+import org.fundacionjala.pivotal.cucumber.selenium.pages.SignIn;
+import org.fundacionjala.pivotal.cucumber.selenium.pages.common.CommonActions;
+import org.fundacionjala.pivotal.cucumber.selenium.pages.common.Navegator;
+import org.fundacionjala.pivotal.cucumber.utils.Environment;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+/**
+ * Created by Administrator on 6/8/2017.
+ */
+public class ToolBars extends AbstractBasePage {
+    private static final int MIN_TIMEOUT = 3;
+    private static final Logger LOGGER = LogManager.getLogger(ToolBars.class);
+
```

Comment:
The static variables should be initialized in a static block, right @carledriss ?